internal/model/fish_buff.go

```diff
@@ -70,6 +70,7 @@ func (f *FishBuff) NextSuggestion() (string, bool) {
 
 // ClearSuggestions clear out all suggestions.
 func (f *FishBuff) ClearSuggestions() {
+	f.suggestions = nil
```

Comment:
Might be best here to keep the allocations for the next set of suggestions vs reset to a nil slice ie f.suggestions = f.suggestions[:0]

Reply:
Thanks for the suggestion ;) made the change